### ISSUE TITLE
Auth Pages - Frontend

### ISSUE THREAD

**Issue by heintayzar-hm** [AHP15/chess-app#7]
- 4 pages => 
- 1. Sign in
- 2. login page
- 3. Forgot password page
- 4. reset password page

Styling => {
 design => https://flowbite.com/blocks/marketing/login/
 Additional => need eyes for password
 form validation => (name, email, password) => library (yup) => javascript
}

Api=> {
   => later
}

**Comment by AHP15** [comment]
@heintayzar-hm, that sounds good, but we should change the logo that they provide.
Are you going to work on it

**Comment by heintayzar-hm** [comment]
Yeah this is easy, I can generate ai logo. from this: https://logo.com/logos/artificial-intelligence

**Comment by AHP15** [comment]
Great, you can assign it to yourself then
